src/starcheck.pl

```diff
@@ -51,13 +51,16 @@
 import perl
 # this provides an interface back to the Perl namespace
 
+try:
```

Comment:
I *thought* I'd stuck the try block in ccd_temp_wrapper because the perl warning array wasn't defined yet if this Inline piece was evaluated first... or something like that.